### PR TITLE
DCS-1720 adding CSV events

### PR DESCRIPTION
**Pull request by michalnawrockidj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/ArrivalsService.kt [hunk 1]
```diff
@@ -46,4 +47,11 @@ class ArrivalsService(
       potentialMatches = matches
     )
   }
+
+  fun getArrivalsAsCsv(startDate: LocalDate, l: Long): String {
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/ArrivalsService.kt:51]: what does the `l` mean?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/confirmedarrival/ConfirmedArrival.kt [hunk 2]
```diff
@@ -12,7 +12,7 @@ import javax.persistence.Table
 
 @Entity
 @Table(name = "confirmed_arrival")
-class ConfirmedArrival(
+data class ConfirmedArrival(
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/confirmedarrival/ConfirmedArrival.kt:15]: `username` on here should be nullable - it probably hasn't mattered in the past as we only ever retrieve arrivals for today but we'll need that to cope with loading historic arrivals

Comment (michalnawrockidj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/confirmedarrival/ConfirmedArrival.kt:15]: sorted

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**